Generate a YAML template and attach it to releases

Eijebong opened the pull request:
Unfortunately we can just `python Launcher.py "Generate Templates"` because archipelago insists on opening windows and then using xdg-open and... So I just made a tiny script that calls the function directly

RePod commented:
> Unfortunately we can just python Launcher.py "Generate Templates"

[WOMM?](https://github.com/RePod/DivaAPworld-R/blob/main/.github/workflows/build.yml#L56)

Eijebong commented:
That'll break the moment something installs xdg-open in your container. I'm curious why it was failing so badly for me though because it was definitely trying to open the launcher and it still does locally if I run your command...

RePod commented:
If the log is anything to go by, `xdg-open` is already available on `ubuntu-latest` but it exhausts all options and still exits gracefully enough for the step to succeed.

Eijebong committed:
Use my workflow to generate releases.

This is basically what the previous workflow did except it also attaches
a template to the release.
